src/lager_transform.erl

```diff
@@ -267,11 +267,8 @@ handle_args(DefaultAttrs, Line, [Arg1, Arg2]) ->
 handle_args(DefaultAttrs, _Line, [Attrs, Format, Args]) ->
     {concat_lists(Attrs, DefaultAttrs), Format, Args}.
 
-make_varname(Prefix, Loc) ->
-    list_to_atom(Prefix ++ atom_to_list(get(module)) ++ integer_to_list(line_from_loc(Loc))).
-
-line_from_loc({Line, _Col}) -> Line;
-line_from_loc(Line) -> Line.
+make_varname(Prefix, CallAnno) ->
```

Comment:
Many more variables should be called `Anno` instead of Line within this file, but I didn't want to pollute the update from its real intention.